eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxWits.hs

```diff
@@ -682,3 +690,9 @@ alonzoEqTxWitsRaw txWits1 txWits2 =
   shelleyEqTxWitsRaw txWits1 txWits2
     && eqRawType (txWits1 ^. datsTxWitsL) (txWits2 ^. datsTxWitsL)
     && eqRawType (txWits1 ^. rdmrsTxWitsL) (txWits2 ^. rdmrsTxWitsL)
+
+nonEmptyListDecodeA :: Decode ('Closed 'Dense) (Annotator x) -> Decode ('Closed 'Dense) (Annotator [x])
+nonEmptyListDecodeA dx = D $ do
```

Comment:
@lehins -  this is the way I wrote it to begin with before you changed it. It feels wrong, but I don't know where else I could check that the collection is not empty. Happy to change it to make it better